### PR TITLE
Sample fix with opencv and openvino

### PR DESCRIPTION
**Pull request by rasapala**

### CODE REVIEW

File: third_party/opencv/opencv_cmake_flags.txt [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
--D WITH_JPEG=ON \
--D WITH_PNG=ON \
--D WITH_TIFF=ON
-
+-DBUILD_LIST=core,improc,imgcodecs,calib3d,features2d,highgui,imgproc,video,videoio,optflow \
+-DOPENCV_EXTRA_MODULES_PATH=/opt/opencv_contrib_repo/modules
+-DCMAKE_BUILD_TYPE=Release \
+-DCMAKE_INSTALL_PREFIX=/opt/opencv \
+-DOPENCV_LIB_INSTALL_PATH=lib \ 
+-DBUILD_TESTS=OFF -DBUILD_PERF_TESTS=OFF -DBUILD_opencv_ts=OFF \
```
Comment (atobiszei) [inline third_party/opencv/opencv_cmake_flags.txt:6]: Why are you changing convention from "-D FLAGNAME" to "-DFLAGNAME and additionally only for some parameters keep the rule one flag per line?

File: src/BUILD [hunk 2]
```diff
@@ -412,7 +411,8 @@ cc_binary(
     ],
     deps = [
         "@google_mediapipe//mediapipe/examples/desktop:simple_run_graph_main",
-        "@mediapipe_calculators//:mediapipe_calculators",
+        #"@mediapipe_calculators//:mediapipe_calculators",
```
Comment (atobiszei) [inline src/BUILD:414]: Why it is commented out?

Comment (rasapala) [reply src/BUILD:414]: Currently one required calculator is commented out in mediapipe, we can enable it later and chenge it to this dependency.

File: demos/common/cpp/Dockerfile [hunk 3]
```diff
@@ -101,7 +101,7 @@ RUN bazel build \
     @com_github_grpc_grpc//:grpc++ \
     @com_google_protobuf//:protobuf_lite \
     @org_tensorflow//tensorflow/core:lib \
-    @opencv//:opencv
+    @linux_opencv//:opencv
```
Comment (dtrawins) [inline demos/common/cpp/Dockerfile:104]: client demos has a separate build configuration from the server so I guess we don't need to change it

**Pull request merged**